stdlib/bytes.mli

```diff
@@ -463,11 +463,37 @@ val of_seq : char Seq.t -> t
 (** Create a string from the generator
     @since 4.07 *)
 
-(** {6 Binary storage of integers} *)
+(** {6 Binary encoding/decoding of integers} *)
 
-(** All following functions raise [Invalid_argument] if the space
+(** The functions in this section decode or encode integers stored
+    in binary form in byte sequences.
+
+    All following functions raise [Invalid_argument] if the space
     needed at index [i] to decode or encode the integer is not
-    available. *)
+    available.
+
+    Little-endian (resp. big-endian) encoding means that least
+    (resp. most) significant bytes are stored first.  Big-endian is
+    also known as network byte order.  Native-endian encoding is
+    either little-endian or big-endian depending on the platform.
```

Comment:
I would link the "depending" to `Sys.big_endian`. That is `{{!Sys.big_endian}depending}`.